R/rt_operational_groups.R

```diff
@@ -1,32 +1,19 @@
-#' Retrieve the list of operational groups used in TaxRef
+#' Retrieve operational groups used in TaxRef
 #'
 #' Operational Groups are names of groups either taxonomic or vernacular
 #' used across TaxRef in French
 #'
```

Comment:
`#' If used without argument returns the list of all operational groups used in TaxRef.`